packages/calling/src/index.ts

```diff
@@ -9,9 +9,10 @@ import Logger from './Logger';
 export {
   createClient,
   createCallHistoryClient,
+  createCallSettingsClient,
+  createContactsClient,
   createMicrophoneStream,
   createVoicemailClient,
-  createContactsClient,
-  createCallSettingsClient,
   Logger,
+  NoiseReductionEffect,
```

Comment:
Do we have to export `NoiseReductionEffect` what is its use exporting?

Reply:
If we don't export, I cannot use it in calling.js

Reply:
I thought developers will use this method to create the NoiseReductionEffect:   static createNoiseReductionEffect(authToken) . What is this about?

Reply:
Yeah, that's what developers are using. But did you see what's happening inside that static function? We are using NoiseReductionEffect inside that function. and to get access of the same, it needs to be re-exported from index.ts

Reply:
I did see. Just noticed the it was a js file. Lack of any return type in the signature and description confused me. Nevermind.